lxd/api_cluster.go

```diff
@@ -1508,8 +1508,10 @@ func updateClusterNode(d *Daemon, r *http.Request, isPatch bool) response.Respon
 	}
 
 	// Update the database
+	changedConfig := []string{}
+	var nodeInfo db.NodeInfo
 	err = d.cluster.Transaction(func(tx *db.ClusterTx) error {
-		nodeInfo, err := tx.GetNodeByName(name)
+		nodeInfo, err = tx.GetNodeByName(name)
```

Comment:
This query isn't actually need AFAIK as its done already on https://github.com/lxc/lxd/pull/10119/commits/66a4a76dae719c65051671ccb79dcec1a5569c76#diff-07fccfa2de6bd472d27c84cb9ca8cd9319e32135864bf114267111df3d3fad2fR1456 so perhaps this can be removed entirely and just use the `node` var above.